### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by Fevix**
typos typos typos im very tired

### CODE REVIEW

**Comment by SigmundGranaas** [comment]
This is looking great! Nice work! I'll try to get a proper look at this after work 👍

File: fabric/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/properties/magnetic.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "type": "patchouli:spotlight",
       "title": "Magnetism",
       "item": "forgero:netherite-refined_pickaxe_head:{FORGERO:{COMPOSITE_TYPE:'SCHEMATIC_PART',CONDITIONS:[],ID:'forgero:netherite-refined_pickaxe_head',INGREDIENTS:[{ID:'forgero:refined_pickaxe_head-schematic',NAME:'refined_pickaxe_head-schematic',NAMESPACE:'forgero',STATE_TYPE:'STATE'},{ID:'minecraft:netherite',NAME:'netherite',NAMESPACE:'minecraft',STATE_TYPE:'STATE'}],NAME:'netherite-refined_pickaxe_head',NAMESPACE:'forgero',STATE_TYPE:'COMPOSITE',TYPE:'PICKAXE_HEAD',UPGRADES:[{ID:'forgero:magnetico-gem',NAME:'magnetico-gem',NAMESPACE:'forgero',STATE_TYPE:'STATE'}]}}",
-      "text": "When the Magnetico Gem is applied to a tool. It's level will determine the range and strength of the magnetic pull of your tool."
+      "text": "When the Magnetico Gem is applied to a tool., its level will determine the range and strength of the magnetic pull of your tool."
```
Comment (SigmundGranaas) [inline fabric/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/properties/magnetic.json]: > "., its"

 I think you meant to remove the period here.

File: fabric/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/wood/crimson_material_entry.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "The crimson fungus, is only found in the nether, and has a distinctive red color. The trunk of this mushroom is actually hard enough to be used in construction of tools."
+      "text": "The crimson fungus is only found in the nether, identiefied by its a distinctive red color. The trunk of this mushroom is actually hard enough to be used in construction of tools."
```
Comment (SigmundGranaas) [inline fabric/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/wood/crimson_material_entry.json]: > identiefied by its a

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/intro/getting_started_tables_entry.json [hunk 3]
```diff
@@ -5,17 +5,27 @@
...
-      "text": "The assembly table is a multi block table which allows you to disassemble and assemble tools. Placing tools or parts in the singular slot, will disassemble it into its components. Be aware, breaking parts into their materials will only yield 1 material. Only tools with 100% durability can be disassembled."
+      "text": "The assembly table is a multi block table which allows you to disassemble tools and parts. Placing tools or parts in the singular slot will disassemble it into its components. Be aware that breaking parts down into their materials will only yield 1 material. Only tools with 100% durability can be disassembled."
     },
     {
       "type": "patchouli:crafting",
       "recipe": "forgero:assembly_table"
+    },
+    {
+      "type": "patchouli:spotlight",
+      "title": "Assembly station",
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/intro/getting_started_tables_entry.json:22]: I assume this was meant to be Upgrade station

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/parts/toolparts_handle_entry.json [hunk 4]
```diff
@@ -5,13 +5,19 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "The handle is an essential part of the tool, but it's stats are less significant than the head. Although the stats are less significant, choosing an unsuitable material for your handle, might affect the usability of your tool, as the weight of the tool might prove to much to be useful"
+      "text": "The handle is an essential part of the tool, but its stats are less significant than the head. Although the stats are less significant, choosing an unsuitable material for your handle might affect the usability of your tool, as the weight of the tool might prove to much to be useful. Some handles also have an additional slot for a pommel"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/parts/toolparts_handle_entry.json]: Missing period at the end.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/properties/weight.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Weight attribute",
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/properties/weight.json:2]: This is a mental note to myself. The weight implementation has changed significantly, so this section is no longer correct. I will need to rewrite it.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/schematics/binding_schematic_entry.json [hunk 6]
```diff
@@ -9,11 +9,11 @@
     },
     {
       "type": "patchouli:crafting",
-      "recipe": "forgero:netherite-binding-schematic_recipe"
+      "recipe": "forgero:netherite-binding"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/schematics/binding_schematic_entry.json:12]: I don't think this recipe exists, but I might also be wrong. It does not follow the fomat of the generated recipes.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/categories/trinkets_category.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Trinkets",
-  "description": "Trinkets are unique artifacts which can be adapted to fit in an upgrade slot of certain tool parts. The trinkets provides unique effect and buffs for your tools",
+  "description": "Trinkets are unique artifacts which can be adapted to fit in an upgrade slot of certain tool parts. The trinkets provide unique effects and buffs for your tools",
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/categories/trinkets_category.json]: Missing period at the end.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/entries/diamond_gem_entry.json [hunk 8]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "The Diamond gem can be applied to all tool parts, and increase the durability the part its applied to"
+      "text": "The Diamond gem can be applied to all tool parts, and increases the durability the part its applied to"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/entries/diamond_gem_entry.json]: Also missing period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/entries/emerald_gem_entry.json [hunk 9]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "The emerald gem increases your durability by 100"
+      "text": "The emerald gem increases your tool's durability by 100"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_gems/en_us/entries/emerald_gem_entry.json]: Also period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/secondary_materials/vines.json [hunk 10]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "Vines can be wrapped around tools and weapons to provide som benefits to handling"
+      "text": "Vines can be wrapped around tools and weapons to provide some benefits to handling"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/secondary_materials/vines.json]: Period at the end.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/copper_material_entry.json [hunk 11]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "Copper is a unique material, as it oxidises over time. It's more rare than iron, and not as hard, making it less useful, but still a good option"
+      "text": "Copper is a unique material, as it oxidises over time. It's more rare than iron, and not as hard, making it less useful but still a good option"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/copper_material_entry.json]: Period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/gold_material_entry.json [hunk 12]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "Gold is the heaviest metal available, and also the softest. It's combined weight and softness makes it a terrible choice for tools. Although not popular as the foundation for tools, it's treasured for it's beauty and rarity. Piglings also don't act hostile if you approach them with equipments embedded with golden upgrades"
+      "text": "Gold is the heaviest metal available, and also the softest. Its combined weight and softness makes it a terrible choice for tools. Although not popular as the foundation for tools, it's treasured for its beauty and rarity. Piglins also don't act hostile if you approach them with equipments embedded with golden upgrades"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/gold_material_entry.json]: Period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/netherite_material_entry.json [hunk 13]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "Netherite is the most durable metal available, but can only be found by mining ancient debree in the nether. Crafting netherite tools might be expensive, as it still requires the same amount of material as other materials to craft parts. Still, the attributes of netherite is unmatched. It's natural association with high heat also makes it resistant to lava and flames"
+      "text": "Netherite is the most durable metal available, but can only be found by mining ancient debris in the nether. Crafting netherite tools might be expensive, as it still requires the same amount of material as other materials to craft parts. Still, the attributes of netherite are unmatched, and its natural association with high heat also makes it resistant to lava and flames"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/metal/netherite_material_entry.json]: Period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/stone/andesite_material_entry.json [hunk 14]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "Andesite is a grey, vulcanic rock commonly found underground in vulcanic areas. It's properties are very similar to normal rocks"
+      "text": "Andesite is a grey, volcanic rock commonly found underground in volcanic areas. It's properties are very similar to normal rocks"
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/stone/andesite_material_entry.json]: Period.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/wood/crimson_material_entry.json [hunk 15]
```diff
@@ -5,7 +5,7 @@
   "pages": [
     {
       "type": "patchouli:text",
-      "text": "The crimson fungus, is only found in the nether, and has a distinctive red color. The trunk of this mushroom is actually hard enough to be used in construction of tools."
+      "text": "The crimson fungus is only found in the nether, identiefied by its a distinctive red color. The trunk of this mushroom is actually hard enough to be used in construction of tools."
```
Comment (SigmundGranaas) [inline forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide_material/en_us/entries/tool_materials/wood/crimson_material_entry.json]: Same comment as previously mentioned.

**Review by SigmundGranaas — COMMENTED** [review]
This is amazing. Thanks for the work! Reading through this was painful, as I had no idea how many errors I made when writing this book. It must have taken so much time to go through everything, so thanks a lot!

I did notice that I must have duplicated some files here and there, and I should probably look into exactly why that happens, and potentially delete files that are not needed. Also you did replace some recipes in the guide, which I am not sure exist in Forgero, so I'll have to boot it up and check if they exist or not.

I did add some comments, but 95% of them were just about missing periods at the end of some sentences, which were also present before, so you don't have to worry about fixing those. 

I also noticed that Git marked some of the property files as entirely new in some cases. I am a little curious about how that happened, but it is not very important to be.

Before this get's merged I want to double check the existence of the recipes you're referring to in the schematic section, but I'll have to do that tomorrow, as it's getting way too late now.

**Comment by SigmundGranaas** [comment]
Oh is there a Forge version of the book as well?

**Comment by Fevix** [comment]
Thanks for the comments

All I really did was update the fabric version of the book and copy paste
that directly over the forge version of the book

I will fix the remaining errors when I get home from work in about 6 hours

**Comment by Fevix** [comment]
> Oh is there a Forge version of the book as well?

There are two versions of the book one in the fabric subfolder and one of the forge subfolder

I just used my computer search function to find all folders named "patchouli"

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/intro/getting_started_tables_entry.json (hunk 3 above)
Comment (Fevix) [reply forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/intro/getting_started_tables_entry.json:22]: This was the original text from the Fabric book.

File: forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/schematics/binding_schematic_entry.json (hunk 6 above)
Comment (Fevix) [reply forgero-forge/src/main/resources/data/forgero/patchouli_books/forgero_guide/en_us/entries/schematics/binding_schematic_entry.json:12]: This was text copied from the Fabric version of the book.

**Comment by SigmundGranaas** [comment]
Ah I see, so that's where the new files were from as well.

**Pull request merged**